ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -805,19 +805,22 @@ Reload RHODS Dashboard Page
 Handle Deletion Confirmation Modal
     [Documentation]    Handles confirmation modal on item deletion
     [Arguments]     ${item_title}    ${item_type}   ${press_cancel}=${FALSE}    ${additional_msg}=${NONE}
+    # Once fixed https://issues.redhat.com/browse/RHODS-9730 change the button xpath to
+    # xpath=//button[text()="Delete ${item_type}"]
+    ${delete_btn_xp}    Set Variable    xpath=//button[contains(text(), 'Delete')]
     Wait Until Generic Modal Appears
     Run Keyword And Warn On Failure    Page Should Contain    Delete ${item_type}?
-    Run Keyword And Continue On Failure    Page Should Contain    This action cannot be undone.
+    Run Keyword And Warn On Failure    Page Should Contain    This action cannot be undone.
     IF    "${additional_msg}" != "${NONE}"
         Run Keyword And Continue On Failure    Page Should Contain    ${additional_msg}
     END
     Run Keyword And Continue On Failure    Page Should Contain    Confirm deletion by typing ${item_title} below:
-    Run Keyword And Continue On Failure    Element Should Be Disabled    xpath=//button[text()="Delete ${item_type}"]
+    Run Keyword And Continue On Failure    Element Should Be Disabled    ${delete_btn_xp}
     Input Text    xpath=//input[@id="delete-modal-input"]    ${item_title}
-    Wait Until Element Is Enabled    xpath=//button[text()="Delete ${item_type}"]
```

Comment:
I would prefer leaving this version of the xpath because it implicitly checks the button text. Naming in RHODS has changed multiple times over time. This implicit check may capture issues with names if they change for some reasons

Reply:
Keeping the current version will fail the new test.
I added a comment
```
    # Once fixed https://issues.redhat.com/browse/RHODS-9730 change the button xpath to
    # xpath=//button[text()="Delete ${item_type}"]
```